src/scrapers/hapoalim.ts

```diff
@@ -209,31 +209,31 @@ async function fetchAccountData(page: Page, baseUrl: string, options: ScraperOpt
   const accounts: TransactionsAccount[] = [];
 
   for (const account of accountsInfo) {
+    debug('getting information for account %s', account.accountNumber);
     let balance: number | undefined;
     const accountNumber = `${account.bankNumber}-${account.branchNumber}-${account.accountNumber}`;
 
     const isActiveAccount = account.accountClosingReasonCode === 0;
     if (isActiveAccount) {
       balance = await getAccountBalance(apiSiteUrl, page, accountNumber);
+      const txns = await getAccountTransactions(
+        baseUrl,
+        apiSiteUrl,
+        page,
+        accountNumber,
+        startDateStr,
```

Comment:
So now we have nothing to do with inactive accounts at all, do you think we even need to return them as empty?

Or maybe, we should find the way to support them? Thinking about that, if it was warking before (is it?) and now it failed, it is a breaking change.

No matter what we decide, I think the code right now should be simpler, since we don't need the inactive account, we should filter it from the loop (the ugliest way will be `if (!isActiveAccount) continue;`, but rearrange the code a little bit to find a better way.